api/types/mount/mount.go

```diff
@@ -108,7 +108,8 @@ type TmpfsOptions struct {
 	SizeBytes int64 `json:",omitempty"`
 	// Mode of the tmpfs upon creation
 	Mode os.FileMode `json:",omitempty"`
-
+	// Whether tmpfs allow exec
+	Exec bool `json:",omitempty"`
```

Comment:
Wondering if this needs a dedicated option, or if we should "punch through" mount options. Bit on the fence for that; pro is that we separate the API from the underlying implementation, but con would be that we may end up having an API change for every mount option that exists.

Reply:
Thank you for the review. I can change that to allow `exec` and any other required options in a generic option passed to Tmpfs mount types.